wandb/sdk/wandb_init.py

```diff
@@ -145,6 +145,12 @@ def setup(self, kwargs):
             self._jupyter_setup(settings)
 
         self._log_setup(settings)
+        # if the wandb dir is not set, set it
+        if os.environ.get(wandb.env.DIR) is None:
+            os.environ["WANDB_DIR"] = os.getcwd()
+        # set settings root dir, for future calls
+        if settings.root_dir is None:
+            settings.root_dir = os.environ["WANDB_DIR"]
```

Comment:
Seems like it would be better to not use env variables to persist internal state here and just make `settings.root_dir` default to `os.getcwd()` if os.environ.get("WANDB_DIR") is None...

Reply:
@raubitsj can you give @KyleGoyette pointers on where to do this in `wandb_settings.py`